llvm/include/llvm/Target/GlobalISel/Combine.td

```diff
@@ -1599,6 +1599,13 @@ def insert_vector_elt_oob : GICombineRule<
          [{ return Helper.matchInsertVectorElementOOB(*${root}, ${matchinfo}); }]),
   (apply [{ Helper.applyBuildFn(*${root}, ${matchinfo}); }])>;
 
+// Combine v8i8 (buildvector i8 (trunc(unmerge)), i8 (trunc), i8 (trunc), i8 (trunc), undef, undef, undef, undef)
+def combine_use_vector_truncate : GICombineRule<
+  (defs root:$root, register_matchinfo:$matchinfo),
+  (match (wip_match_opcode G_BUILD_VECTOR):$root,
```

Comment:
Don't use `wip_match_opcode` anymore. See https://github.com/llvm/llvm-project/blob/main/llvm/include/llvm/Target/GlobalISel/Combine.td#L1863 for an example using `G_BUILD_VECTOR` and `GIVariadic`.

Reply:
Why did you `wip_match_opcode G_BUILD_VECTOR`when there is no build vector in your summary?